packages/web/src/validators/checkbox-validators/checkbox-validators.ts

```diff
@@ -4,5 +4,5 @@ export const requiredCheck: Validator<boolean> = {
     validate: (value: boolean) => {
         return value;
     },
-    errorMessage: { "en": "Please complete the required field to continue.",  "fr": "Veuillez compléter les champs obligatoires afin de continuer." }
+    errorMessage: { "en": "You must check the box to continue.",  "fr": "Vous devez cocher la case pour continuer." }
```

Comment:
Can you move the `en` and the `fr` each to its own line to make it easier to read?